apps/src/aichat/views/AichatView.tsx

```diff
@@ -3,11 +3,24 @@
 import React from 'react';
 import moduleStyles from './aichatView.module.scss';
 import ChatWorkspace from './ChatWorkspace';
+import Instructions from '@cdo/apps/lab2/views/components/Instructions';
+import PanelContainer from '@cdo/apps/lab2/views/components/PanelContainer';
+const commonI18n = require('@cdo/locale');
 
 const AichatView: React.FunctionComponent = () => {
   return (
     <div id="aichat-lab" className={moduleStyles.aichatLab}>
-      <ChatWorkspace />
+      <div className={moduleStyles.instructionsArea}>
+        <PanelContainer
+          id="aichat-instructions"
+          headerText={commonI18n.instructions()}
+        >
+          <Instructions />
+        </PanelContainer>
+      </div>
+      <div className={moduleStyles.chatWorkspaceArea}>
+        <ChatWorkspace />
```

Comment:
`ChatWorkspace` includes the `PanelContainer` and I see here that `Instructions` does not. Should we make this consistent in either direction?
I can update `ChatWorkspace` if that makes more sense.

Reply:
Hmm, I don't have a strong preference - I think if we want to make it consistent then yeah moving `PanelContainer` out of `ChatWorkspace` would make more sense? I'd want to avoid putting a `PanelContainer` inside of `Instructions` since it's meant to be a reusable component.